includes/services.inc.php

```diff
@@ -129,7 +129,7 @@ function poll_service($service)
 
     // If we do not have a cmd from the check script, build one.
     if ($check_cmd == "") {
-        $check_cmd = $config['nagios_plugins'] . "/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);
```

Comment:
I created new check file /includes/services/check_ping_custom.inc.php

`<?php

$check_cmd = $config['nagios_plugins'] . "/check_ping ".$service['service_param'];
`
How to use it in LibreNMS? I don't see it in the services list